app/views/organizations/organization_profiles/_form.html.erb

```diff
@@ -4,57 +4,105 @@
-                              'data-initial-value': location.object.country,
-                              required: true,
-                              class: 'form-control' %>
-          <% profile.location.custom_messages(:country).each do |error| %>
-            <div class="alert alert-danger mt-1" role="alert"><%= error.message %></div>
-          <% end %>
+    <!-- Administrative details section -->
+    <div class="row mt-3">
+      <!-- First Column -->
+      <div class="col-lg-6">
+        <%= form.fields_for :organization do |org_form| %>
+          <form class="mb-3">
+            <!-- Phone Number -->
+            <div class="form-group">
+              <%= form.telephone_field :phone_number,
+                                      autofocus: true,
+                                      placeholder: "10 digit number",
+                                      class: 'form-control' %>
+            </div>
 
-          <%= location.select :province_state,
-                              CS.states(location.object.country).invert,
-                              { prompt: "Please select" },
-                              'data-country-state-target': 'state',
-                              required: true,
-                              class: 'form-control' %>
-          <% profile.location.custom_messages(:province_state).each do |error| %>
-            <div class="alert alert-danger mt-1" role="alert"><%= error.message %></div>
-          <% end %>
+            <!-- Email -->
+            <div class="form-group">
+              <%= form.text_field :email, placeholder: "john@email.com", class: 'form-control' %>
+            </div>
+            
+            <!-- About Us -->
+             <div class="form-group">
+              <%= form.text_field :about_us, placeholder: "We get pets into loving homes!", class: 'form-control' %>
+            </div>
+          </form>
+        <% end %>
+      </div>
+
+      <!-- Second Column -->
+      <div class="col-lg-6">
+
+        <!-- Nested Form for Locations Table -->
+        <div class="form-group bigger" data-controller="country-state">
+          <%= form.fields_for :location do |location| %>
+            <!-- Country Select -->
+            <%= location.select :country,
+                                CS.countries.invert,
+                                { prompt: "Please select" },
+                                'data-path': country_states_path(:country),
+                                'data-country-state-target': 'country',
+                                'data-action': 'change->country-state#updateStates',
+                                'data-initial-value': location.object.country,
+                                required: true,
+                                class: 'form-control' %>
+            <% profile.location.custom_messages(:country).each do |error| %>
+              <div class="alert alert-danger mt-1" role="alert"><%= error.message %></div>
+            <% end %>
```

Comment:
Not 100% sure but I think this isn't needed, doesn't bootstrap form handle showing the errors?